src/main/resources/log4j2.properties

```diff
@@ -0,0 +1,10 @@
+name = STConfig
+appender.console.type = Console
+appender.console.name = STDOUT
+appender.console.layout.type = PatternLayout
+appender.console.layout.pattern = %d{yyyy-MM-dd HH:mm:ss}{GMT} [%thread] %highlight{%-5p} [%c{1}:%L] %m%n
+
+rootLogger.level = DEBUG
+rootLogger.appenderRef.console.ref = STDOUT
+rootLogger.appenderRef.console.level = ${env:STRIMZI_TEST_LOG_LEVEL:-INFO}
+rootLogger.additivity = false
```

Comment:
Well this is needed for log4j2. And because we don't want have `systemtest` module (and new poms etc.) and the log4j2 is looking on the `main->resources->log4j2.properties` path, I put it there. But I guess that I can add it somewhere else and point some env var to the file location.

Reply:
If it is used in tests, shouldn't it be in the test resources? If it is needed at runtime, is DEBUG rely what we want? And do we want the `env:STRIMZI_TEST_LOG_LEVEL`? Is it even used at runtime with native build?